aggregates/src/commonMain/kotlin/org/dhis2/mobile/aggregates/data/DisableDataElementGrouping.kt

```diff
@@ -0,0 +1,32 @@
+package org.dhis2.mobile.aggregates.data
+
+import org.dhis2.mobile.aggregates.model.CellElement
+
+internal object DisableDataElementGrouping {
```

Comment:
is this like a mapping inside the data layer? could it be a extension function or a mapper, can you explain the functionality?